### ISSUE TITLE
Transactions are not deleted after export if the flag is set via settings

### ISSUE THREAD

**Issue by realrosid** [GnuCash-Pocket/gnucash-android#151]
## Steps to reproduce the behaviour
1. Activate "Delete all transactions after export" in settings -> Backup & export
2. Enter any transaction
3. Export transaction. The flag "Delete transactions after export" is set automatically
4. Check transactions

## Expected behaviour
The transaction should be deleted.

## Actual behaviour
The transaction is not deleted.
The only way to have the transactions deleted is by toggling the flag manually. If the flag is set for all exports via settings, deleting does not work.

## Software specifications
* GnuCash Android version: 2.5.1
* System Android version: 14
* Device type: Xiaomi 14

**Comment by FlavioB79** [comment]
For me there's no way making it work. No workaround :-(
Can this be investigated deeply?
In the meantime: how do I proceed to empty all transactions anyways (I've already exported them)?

**Comment by realrosid** [comment]
You are right, the workaround does not work anymore. You can delete all transactions by going into settings -> transactions -> delete all transactions
